processor/src/main/java/org/mapstruct/ap/internal/model/source/SourceMethod.java

```diff
@@ -376,12 +349,6 @@ public boolean isMapMapping() {
         return isMapMapping;
     }
 
-    public boolean isEnumMapping() {
```

Comment:
Why not keep the method?

Reply:
I see that we are still doing a `isEnumMapping` check. So maybe we should remove everywhere and if someone is using `@Mapping` instead of `@Value` throw an error?

Reply:
Good idea. I did not search actively for the check. I'll do so and throw an error at top level.